packages/plugin-blockquote/src/BlockquoteDomParser.ts

```diff
@@ -1,11 +1,11 @@
 import { AbstractParser } from '../../plugin-parser/src/AbstractParser';
-import { DomParsingEngine } from '../../plugin-dom/src/DomParsingEngine';
+import { XmlParsingEngine } from '../../plugin-xml/src/XmlParsingEngine';
 import { BlockquoteNode } from './BlockquoteNode';
 import { nodeName } from '../../utils/src/utils';
 
 export class BlockquoteDomParser extends AbstractParser<Node> {
```

Comment:
All these `<...>Domparser` should be renamed `<...>XmlParser` if they are supposed to go into the `XmlParsingEngine`.